palindrome.c

```diff
@@ -0,0 +1,18 @@
+#include <stdio.h>
+#include <string.h>
```

Comment:
nice job, but you didn't properly handle some of the test cases - you didn't handle the empty string (which is a palindrome) and didn't take capitals into consideration!